src/execution.jl

```diff
@@ -143,7 +143,7 @@ macro roc(ex...)
                 local queue = get_default_queue(agent)
                 local signal = HSASignal()
                 kernel(queue, signal, kernel_args...; $(map(esc, call_kwargs)...))
-                wait(signal)
+                #wait(signal)
```

Comment:
Commented out so we don't hang while running tests, which makes it easier to tease out multiple issues at once.

Reply:
Reminder to myself to fix this!